### DISCUSSION TITLE
Build Checks failing with Pull Request

### DISCUSSION THREAD

**Discussion by Roshni-P** [zealdocs/zeal#1675] [Q&A] [answered]
My Pull Request #1673 fails in most of the Build checks. I had forked the repo and thereafter created the pull request.
The error that I see is - "Error: "'/usr/local/bin/cmake' failed with error code: '1'.".

Any ideas on how I could resolve them?

**Comment by trollixx** [comment, marked as answer]
There's an issue with the code that triggers the build failure. I commented on the pull request.
